### ISSUE TITLE
Adjustments to the jobSchema

### ISSUE THREAD

**Issue by coderjake91** [AshisPatel/excella#14]
[ done] 1. In the contactSchema (sub-doc), adjust name field to be two separate fields (lastName and firstName)

[ done] 2. Add a 'deleteContact' resolver in addition to the existing 'updateContact' and 'addContact' resolvers

[ done] 3. Change typeDef/resolver for 'updateContact' to accept a Contact _id and return the updated contact

[done ] 4. Change the existing date field (application Date) in the jobSchema to 'lastUpdated'
[done ] 5. Add a query 'singleJob' to the existing resolvers, that accepts a current Job's _id, and returns a single Job

**Comment by coderjake91** [comment]
_**TL:DR - > we will need to format incoming user date data either on the front-end or back-end, in order for the data to be properly stored with how our typeDefs are currently.**_

With respect to Item 4 above, we will need some form of user input date validation that can occur either on the front-end or the back-end.

currently: if the user enters a date into whatever field exists on the client-side (i.e. 11/28/2021) the date is stored converted to the date-formate through the mutation resembling  "lastUpdated": "2021-11-28T06:00:00.000Z"

please note that invalid mutations requested through the server (such as 11/2021, MM/YYYY) will return with an error message in graphql:      "message": "Cast to date failed for value \"11/2021\" (type string) at path \"lastUpdated\""

The user  date input will have to be converted to a format that resembles one of the RFC 3339 scalars that the referenced npm package (below) talks about...

**RFC 3339 "defines a date and time format for use in Internet protocols that is a profile of the ISO 8601 standard for representation of dates and times using the Gregorian calendar."**

Currently our graphql schema has a custom 'Date' scalar in the typeDefs, and uses the  [graphql Date scalar](https://www.graphql-scalars.dev/docs/scalars/date-time) format, and the [npm package](https://www.npmjs.com/package/graphql-iso-date) to format our dates entering the data-base through the mutation endpoint, in the MongoDB schema, the 'lastUpdated' field is of type: Date, and defaults to the current date using Date.now (JavaScript date constructor), the getter method is currently not active that uses the dateFormat method in /utils.
